src/Documentation/SidebarMenu/SidebarMenuHelper.js

```diff
@@ -0,0 +1,161 @@
+import startCase from 'lodash.startcase'
+import kebabCase from 'lodash.kebabcase'
+import sidebar from '../sidebar'
+
+export const PATH_SEPARATOR = '/'
+export const PATH_TO_DOC = '/doc'
+
+export default class SidebarMenuHelper {
+  static fillFilesArray(section, file, arr) {
+    let folder = SidebarMenuHelper.getParentFolder(file, section)
+    let filename = SidebarMenuHelper.extractFilename(file)
+    let path = SidebarMenuHelper.getFullPath(folder, filename)
+    arr[path] = startCase(
+      SidebarMenuHelper.removeExtensionFromFileName(filename)
+    )
+  }
+
+  static isString = obj => typeof obj === 'string'
+
+  static transformAbsoluteToDocRelatedPath = path =>
+    path
+      .join(PATH_SEPARATOR)
+      .replace(new RegExp(PATH_TO_DOC, 'i'), '')
+      .split(PATH_SEPARATOR)
+      .filter(_ => _)
+
+  static combineToPath = subPaths => [].concat(subPaths).join(PATH_SEPARATOR)
```

Comment:
just curious - why is `[].concat(subPaths)` needed? subPath is always non empty array, right?

Reply:
It will make function safe, exception won’t be thrown even if `subPaths` is` null` or `undefined`. May be it’s too much, agreed.